extensions/positron-catalog-explorer/src/extension.ts

```diff
@@ -0,0 +1,46 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (C) 2025 Posit Software, PBC. All rights reserved.
+ *  Licensed under the Elastic License 2.0. See LICENSE.txt for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import {
+	CatalogProviderRegistry,
+	registerCatalogCommands,
+	registerTreeViewProvider,
+} from './catalog';
+import { DefaultDatabricksCredentialProvider } from './credentials';
+import { registerDatabricksProvider } from './catalogs/databricks';
+import { registerMockProvider } from './catalogs/mock';
+import { registerDbfsProvider } from './fs/dbfs';
+import { setExtensionUri } from './resources';
+
+export async function activate(context: vscode.ExtensionContext) {
+	// Check if the extension is enabled via configuration
+	const config = vscode.workspace.getConfiguration('positronCatalogExplorer');
```

Comment:
Should we remove the word `positron` from this config group? I think we were trying to keep that out of settings.